src/widgets/TokenAnnotationWidget.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <table class="selected-tokens">
+    <tr colspan="3" v-if="selectedToken">
+      <a href @click.prevent="onClear">Clear Filter</a>
+    </tr>
+    <tr v-for="token in tokens" :key="token.veRef">
+      <th class="text">{{ token.value }}</th>
+      <td class="parse">{{ token.tag }}</td>
+      <td class="text">{{ token.lemma }}</td>
+    </tr>
+  </table>
+</template>
+
+<script>
+  import gql from 'graphql-tag';
+  import { URN } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS, CLEAR_TOKEN } from '@/reader/constants';
+
+  export default {
+    scaifeConfig: {
+      displayName: 'Token Annotations',
+    },
+    methods: {
+      prettyPrint(token) {
+        return JSON.stringify(token, null, 2);
+      },
+      onClear() {
+        this.$store.dispatch(`${MODULE_NS}/${CLEAR_TOKEN}`);
+      },
+    },
+    computed: {
+      selectedToken() {
+        return this.$store.state[MODULE_NS].selectedToken;
+      },
+      // TODO: Dedupe from Reader.vue
+      urn() {
+        return this.$route.query.urn
+          ? new URN(this.$route.query.urn)
+          : this.$store.getters[`${MODULE_NS}/firstPassageUrn`];
+      },
+      // TODO: Dedupe from Reader.vue
+      gqlQuery() {
+        if (this.urn) {
+          return gql`
+          {
+            passageTextParts(reference: "${this.urn.absolute}") {
+              edges {
+                node {
+                  id
+                  ref
+                  tokens {
+                    edges {
+                      node {
+                        veRef
+                        value
```

Comment:
`wordRef` would probably be better here (will omit punctuation)

Reply:
I get a `400` when trying to use `wordRef`

Reply:
<img width="467" alt="Screen Shot 2020-05-14 at 9 44 11 PM" src="https://user-images.githubusercontent.com/1192/82005848-11d32880-962c-11ea-94d6-e2c5c32e5b88.png">

Reply:
I think `wordValue` is what you meant.  that seems to do it.